core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
```

Comment:
I guess, global non scoped lookups for directives are non idea, I wonder if there is a better solution than just grabbing the global bk-notebook DOM node?